extensions/commands/art/cmd_build_info.py

```diff
@@ -173,7 +173,7 @@ def _get_local_artifacts():
         def _get_remote_artifacts(artifact):
             artifact_info = None
             assert self._url and self._repository, "Missing information in the Conan local cache, " \
-                                                   "please provide the --url and --repository arguments " \
+                                                   "please provide '--url' or '--server' arguments " \
```

Comment:
I think we also want to change the assert above to not check the self._repository (although it should always be there...)